Scripts/Player_Scripts/player.gd

```diff
@@ -76,3 +84,16 @@ func crouch():
 			b_is_crouching = false
 			current_move_speed = walking_speed
 			animation_player.play_backwards("crouch")
+			
+func interact_with_object():
+	if Input.is_action_just_pressed("primary_action"):
+		interaction_component.interact_with_item()
+
+func drop_held_object():
+	if Input.is_action_just_pressed("secondary_action"):
+		interaction_component.drop_item()
+		
+func throw_held_object():
+	if Input.is_action_just_pressed("throw"):
```

Comment:
this is bound to E, but E is already primary action.

Please edit this to use "secondary action" or remove E from primary and set it to another key,

Reply:
or even rename secondary action

Reply:
primary action on my branch was LMB